.ado/templates/react-native-init.yml

```diff
@@ -8,8 +8,6 @@ parameters:
   vsComponents: ''
   listVsComponents: false
   installVsComponents: false
-
-variables:
   BuildLogDirectory:  $(Build.BinariesDirectory)\$(BuildConfiguration)\$(BuildPlatform)\BuildLogs
```

Comment:
@dannyvv did you add this parameter `BuildLogDirectory`?
If so, why use Configuration\Platform instead of Platform\Configuration as used in the rest of build process?

Reply:
No reason, other than habbit I'll flip it.

Reply:
Actually it turns out you can't use variables and when using parameters, there is no variable replacement in the default value, so I had to spell it out without reuse :( for react-native-init...